src/css/base/typography.scss

```diff
@@ -4,7 +4,23 @@
 body {
   color: $color-base;
   font-weight: 400;
-  line-height: 1.7333333;
+  line-height: $leading-body;
+  font-size: $base-type-px;
+}
+
+a {
+  text-decoration: none;
```

Comment:
no, they have an underline, but it's a border-bottom — at least in the docs, gotta do more scoping/spec'ing to make it work properly in the dashboard